docs/monitoring/spm-faq.md

```diff
@@ -48,7 +56,7 @@ Otherwise, restart your application/java process.
 **How frequently does SPM monitor collect metrics and can I adjust that interval?**
 
 SPM monitor by default collects metrics every 10 seconds. To reduce this
-frequency to 30 seconds, for example, simply add the following line to
+frequency to 30 seconds, for example, for Java agent simply add the following line to
```

Comment:
Do we care to say *Java* agent?  It's just written in Java.  We are not calling STA a Go agent.

Reply:
I started with Java agent and Infra agent as names for the individual agent. After sync with Bojan we decided to go with `sematext-agent-java` and `st-agent`. The latest docs uses this names.